.github/workflows/publish-to-maven.yml

```diff
@@ -35,3 +35,21 @@ jobs:
           PGP_SECRET: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+
+    # Publish Released Fat Jar to Blob Storage 
+      - name: Gradle build
+        run: |
+          ./gradlew build
+          # remote folder for CI upload
+          echo "CI_SPARK_REMOTE_JAR_FOLDER=feathr_jar_release" >> $GITHUB_ENV
+          # get local jar name without path
+          echo "FEATHR_LOCAL_JAR_FULL_NAME_PATH=$(ls build/libs/*.jar)" >> $GITHUB_ENV
+      
+      - name: Azure Blob Storage Upload (Overwrite)
+        uses: fixpoint/azblob-upload-artifact@v4
+        with:
+          connection-string: ${{secrets.SPARK_JAR_BLOB_CONNECTION_STRING}}
+          name: ${{ env.CI_SPARK_REMOTE_JAR_FOLDER}}
+          path: ${{ env.FEATHR_LOCAL_JAR_FULL_NAME_PATH}}
```

Comment:
Suggest change 

- `CI_SPARK_REMOTE_JAR_FOLDER` => `FEATHR_JAR_AZURE_STORAGE_FOLDER`
- `FEATHR_LOCAL_JAR_FULL_NAME_PATH` => `FEATHR_JAR_LOCAL_FULL_PATH`

Reply:
Those are the usage practice copied from https://github.com/feathr-ai/feathr/blob/main/.github/workflows/pull_request_push_test.yml, do you think I should update both?

Reply:
I didn't realize it's already in use, then let's avoid change names already in use.